Core/functions.php

```diff
@@ -205,22 +201,22 @@ function getOptionFromInteraction(Collection|Interaction $options, string ...$na
 
 function log($level, string $message, array $context = []): void
 {
-    env()->discord->getLogger()->log($level, $message, $context);
+    discord()?->getLogger()->log($level, $message, $context);
 }
 
 function debug(string $message, array $context = []): void
 {
-    env()->discord->getLogger()->debug($message, $context);
+    discord()?->getLogger()->debug($message, $context);
 }
 
 function error(string $message, array $context = []): void
 {
-    env()->discord->getLogger()->error($message, $context);
+    discord()?->getLogger()->error($message, $context);
 }
 
 function info(string $message, array $context = []): void
 {
-    env()->discord->getLogger()->info($message, $context);
+    discord()?->getLogger()->info($message, $context);
 }
```

Comment:
Calling env directly so there's not two function calls (I should've done `Env::get()` directly tbh)

Reply:
Well, actually, I was thinking that `discord()` function should throw if `env()->discord` is null, but in the end I decided that this change is beyond the scope of current PR and left it out.